config/prod.json

```diff
@@ -1,4 +1,4 @@
 {
-  "project": "terra-bard-prod",
+  "project": "terra-bard",
   "samRoot": "https://sam.dsde-prod.broadinstitute.org"
```

Comment:
since this file is the same as `config.json`, do we want to do a symbolic link instead?

Reply:
config.json was accidentally added - thanks for pointing this out!